css-two-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  max-width: 900px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: Karla, sans-serif;
+}
+
+.column-full {
+  text-align: center;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-half {
+  width: 50%;
+  padding-top: 30px;
+  padding-bottom: 30px;
```

Comment:
Remember, `column` classes should only be `width` or `flex-basis`. You should move these `padding` declarations to their own separate class that you can add to the element in addition to the `column` class.

Reply:
Thank you, I will make that change.